test/Tests/src/Semantic/Conversion_Spec.enso

```diff
@@ -112,4 +112,7 @@ spec =
             meta_from.rename "foo" 123 . should_equal "foo called"
             meta_from.rename "foo" . should_equal .foo
 
+        Test.specify "should not allow currying" <|
+            Panic.recover Any (Foo.from) . catch Any .to_display_text . should_equal "Conversion currying without `that` argument is not supported."
```

Comment:
Where is this `Conversion currying without ...` message coming from? I don't see it defined in the code anywhere now. 🪄